hype-submitter/src/main/java/com/spotify/hype/model/PersistentDisk.java

```diff
@@ -0,0 +1,38 @@
+/*-
+ * -\-\-
+ * hype-submitter
+ * --
+ * Copyright (C) 2016 - 2017 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.hype.model;
+
+import io.norberg.automatter.AutoMatter;
+
+/**
+ * Request an existing Google Compute Engine (GCE) Persistent Disk into your pod.
+ *
+ * <p>see https://kubernetes.io/docs/concepts/storage/volumes/#gcepersistentdisk
+ */
+@AutoMatter
+public interface PersistentDisk extends Volume {
```

Comment:
If this only applies to a GCE persistent disk this should be renamed to `GCEPersistentDisk`